MTGToolbox.Repository/CardRepository.cs

```diff
@@ -1,52 +1,32 @@
-﻿using System;
+﻿using Microsoft.EntityFrameworkCore;
+using MTGToolbox.Abstract;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
-using MTGToolbox.Core;
 
 namespace MTGToolbox.Repository
 {
-    public class CardRepository : ICardRepository, IDisposable
```

Comment:
Also optional, but I removed the IDisposable because the only thing that needs to be disposed is the DbContext. And that's automatically handled by ASP.NET Core's dependency injection. Once the request has completed all of its work, ASP.NET Core's DI will dispose anything that implements IDisposable for us.